lottie/src/main/java/com/airbnb/lottie/LottieImageAsset.java

```diff
@@ -25,7 +27,7 @@ private Factory() {
 
     static LottieImageAsset newInstance(JSONObject imageJson) {
       return new LottieImageAsset(imageJson.optInt("w"), imageJson.optInt("h"), imageJson.optString("id"),
-          imageJson.optString("p"));
+          imageJson.optString("p"), imageJson.optString("u"));
```

Comment:
What is this?

Reply:
@gpeal he added fifth argument (check line 16.) called `dirName` with value of `imageJson.optString("u")`.